nonthermal.cc

```diff
@@ -467,7 +467,64 @@ void init(const int my_rank, const int ndo_nonempty) {
     fflush(nonthermalfile);
   }
 
+  int LIST_LENGTH = MAX_NT_EXCITATIONS_STORED;
```

Comment:
What is the purpose of this alias?

Reply:
Was part of the printouts I had for checking allocations etc. I will remove / clean up the code